vnext/Microsoft.ReactNative/Utils/ResourceBrushUtils.cpp

```diff
@@ -35,12 +31,21 @@ void UpdateResourceBrush(
     const std::wstring &resourceName,
     const xaml::Media::Brush brush) {
   const auto resources = element.Resources();
-  if (resources != nullptr) {
-    if (brush != nullptr) {
-      resources.Insert(winrt::box_value(resourceName), brush);
-    } else {
-      resources.Remove(winrt::box_value(resourceName));
-    }
+
+  // sanity check for null pointers
+  if (resources != nullptr && brush != nullptr) {
```

Comment:
It looks like the previous behavior of a null `brush` was to remove the brush from the resource, but after the change we no-op. Would this affect any other callers of `UpdateResourceBrush`.

Reply:
good catch, removing this line didn't cause any problems but in further inspection TextInput would have to make changes to use my new version of UpdateResourceBrush. I split my version to another function, UpdateToggleResourceBrush.